enterprise/frontend/src/embedding-sdk/conventional-changelog-config.js

```diff
@@ -1,4 +1,51 @@
 // eslint-disable-next-line import/no-commonjs
 module.exports = {
-  gitRawCommitsOpts: { grep: "(sdk)" },
+  gitRawCommitsOpts: {
+    grep: "(sdk)",
+  },
+  options: {
+    preset: "angular",
```

Comment:
❤️

Reply:
🤣 the recommended preset is Angular. But it doesn't have anything to do with Angular, it's just how Angular does changelog.